starter-code/routes/celebrity.routes.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express");
+const router = express.Router();
+require("dotenv").config();
+const CelebrityModel = require("../models/celebrity.model");
+
+router.get("/", (req, res) => {
+  CelebrityModel.find().then((celeb) => {
+    res.render("celebrity.hbs", { celeb });
+  });
+});
+
+// celebrity/:id
+router.get("/showdetails/:id", (req, res) => {
+  let id = req.params.id;
+  CelebrityModel.findById(id)
+  .then((celeb) => {
+    res.render("show.hbs", { celeb });
+  });
+});
+
+// celebrities/new
+router.get('/new', (req, res, next) => {
+      res.render('new.hbs');
+    });
+
+  router.post("/new", (req, res) => {
+        CelebrityModel.create(req.body)
+          .then((celeb) => {
```

Comment:
since you are not doing anything with the celeb parameter information then you don't need to put it.